README.md

```diff
@@ -18,29 +18,29 @@
   </a>
 </p>
 
-Sunmao(榫卯 /suən mɑʊ/) is a front-end low-code framework. Through Sunmao, you can easily encapsulate any front-end UI components into low-code component libraries, so as to build your own low-code UI development platform, making front-end development as tight as Sunmao.
+Sunmao(榫卯 /suən mɑʊ/) is a front-end low-code framework. Through Sunmao, you can easily encapsulate any front-end UI components into low-code component libraries to build your own low-code UI development platform, making front-end development as tight as Sunmao("mortise and tenon" in Chinese).
 
 [中文](./docs/zh/README.md)
 
 ## DEMO
 
-Sunmao‘s website is developed with Sunmao, look here: [Sunmao website editor](https://sunmao-ui.com/dev.html)
+The offcial website of Sunmao is developed by Sunmao, try it from here: [Sunmao website editor](https://sunmao-ui.com/dev.html)
 
 We also provide an open-to-use template: [Sunmao Starter Kit](https://github.com/webzard-io/sunmao-start)
 
 ## Why Sunmao?
 
-### Reactive low-code framework
+### Responsive low-code framework
```

Comment:
In frontend development, the word 'responsive' is mainly used to describe a website that can re-layout/resize when the device's screen size changes.

And the word 'reactive' means the view will automatically be updated when the state changes, which is a programming concept more than a UX.
An example quote from Vue.js's official website:
```
Performant
Truly reactive, compiler-optimized rendering system that rarely requires manual optimization.
```

But yes, here we'd better say `Reactive rendering` or `Reactive state (store/management)`.

Reply:
cc @tanbowensg